library/core/src/array/mod.rs

```diff
@@ -364,3 +364,54 @@ macro_rules! array_impl_default {
 }
 
 array_impl_default! {32, T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T}
+
+#[cfg(not(bootstrap))]
+#[lang = "array"]
+impl<T, const N: usize> [T; N] {
+    /// Returns an array of the same size as self, with `f` applied to each element.
+    ///
+    /// # Examples
+    /// ```
+    /// #![feature(array_map)]
```

Comment:
```suggestion
    /// # #![feature(array_map)]
```

Should we hide this? rustdoc already reports it.